benchmark/src/main/scala/spire/benchmark/ComplexAddBenchmarks.scala

```diff
@@ -1,14 +1,14 @@
 package spire
 package benchmark.jmh
-import java.util.concurrent.TimeUnit
-
-import scala.util.Random
-import Random._
-import org.openjdk.jmh.annotations._
-import spire.algebra._
-import spire.math._
-import spire.implicits._
-import spire.benchmark.Arrays.init
+// import java.util.concurrent.TimeUnit
+//
+// import scala.util.Random
+// import Random._
+// import org.openjdk.jmh.annotations._
+// import spire.algebra._
+// import spire.math._
+// import spire.implicits._
+// import spire.benchmark.Arrays.init
 
 /*
```

Comment:
Have you tried compiling this benchmark?

Reply:
Indeed, there are several commented out benchmarks. Maybe I can do that in a separate PR and if I can't just delete them?

Reply:
Sounds good :+1: